osconfig_tests/main.go

```diff
@@ -15,37 +15,17 @@
 package main
```

Comment:
We should not have cross-dependencies between OS Config and image/OVF import/export code, even for tests. OS Config will most likely be moved to a different repo in the near future. 

@adjackura  can provide more context

Reply:
I agree. Here it's not a cross dependency. They both depend on test_common, which is not specific for any feature and can be copied if we really need to split the codebase.

Reply:
Andrew mentioned osconfig will have more specific refactoring, so I will make a common lib for only ovf + image import/export